modules/claims_api/config/schemas/v1/2122.json

```diff
@@ -30,7 +30,12 @@
               "pattern": "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$",
               "maxLength": 30
             },
-            "aptUnitNumber":   { "type": "integer" },
+            "aptUnitNumber": {
+              "oneOf": [
+                { "type": "integer" },
+                { "type":"string" }
+              ]
+            },
             "city": {
               "description": "City for the address.",
```

Comment:
@mchristiansonVA this seems weird to me, like the city and aptUnitNumber are one attribute
![Uploading Screenshot 2024-06-17 at 11.38.55 AM.png…]()

Reply:
I neglected to add the generated swagger, so latest updates should show this better. Note that the screenshot shows the original implementation, which doesn't have space between the aptUnitNumber and city attributes.